client/src/components/ProductDetails/StyleSelector.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import css from './prodStyles.css';
+import StyleThumbnail from './StyleThumbnail.jsx'
+
+//props(styles) = array of objs. ea obj is a style
```

Comment:
Did you mean to leave this comment in the final code? It might not be needed since you can understand it from the code.

Reply:
(same for other similar comments around the PR)

Reply:
I just deleted the comments left in all files. I copied them into a google docs for my personal notes